src/lexitools/commands/correspondences.py

```diff
@@ -1,161 +1,373 @@
 """
-Check the prosodic structure of a given dataset.
+Extract correspondences from a set of datasets.
 """
+from dataclasses import dataclass, asdict
 
 from clldutils.clilib import add_format
 from cldfbench.cli_util import add_catalog_spec
 
 from pyconcepticon.api import Concepticon
 from cldfcatalog import Config
 
-# lingpy for alignments
 import lingpy
-
 from itertools import combinations, product
 from collections import Counter, defaultdict
 from pylexibank import progressbar
 import pyglottolog
-import networkx as nx
 import csv
 import csvw
 import time
 import subprocess
 from importlib.util import find_spec
 from cldfbench import get_dataset
+import sys
+import json
+from lexitools.coarse_soundclass import Coarsen
+from typing import List
+
+LEXICORE = [('lexibank', 'aaleykusunda'), ('lexibank', 'abrahammonpa'),
```

Comment:
Hmm, we should probably store this list somewhere else rather than in code. @LinguList or @xrotwang, thoughts ? 
Maybe we could just pass in a csv of datasets for this type of command?

Reply:
This is a (very) temporary fix, waiting for the proper Lexicore interface. This and the MockLexicore class are meant to be removed from the code entirely and replaced with a few calls to the lexicore package (when it is ready).